.flowconfig

```diff
@@ -1,5 +1,5 @@
 [version]
-^0.115.0
+^0.119.1
```

Comment:
I don't know enough about our flow usage, but I would prefer not to change the version in this PR 😄

Reply:
Neither do i   😄, i just asked @daern91  and he told me to pump it

Reply:
I see 👍